service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/handlers/DraftOrdersApprovedEmailTemplateTest.java

```diff
@@ -0,0 +1,146 @@
+    NotificationService.class,
+    ObjectMapper.class,
+    CaseUrlService.class,
+    RepresentativeNotificationService.class,
+})
+class DraftOrdersApprovedEmailTemplateTest extends EmailTemplateTest {
+
+    @Autowired
+    private DraftOrdersApprovedEventHandler underTest;
+
+    @MockBean
+    private CoreCaseDataService coreCaseDataService;
+
+    @MockBean
+    private CafcassLookupConfiguration cafcassLookupConfiguration;
+
+    @Test
+    void notifyLA() {
+
+        Element<HearingBooking> hearingBooking = element(HearingBooking.builder()
+            .type(HearingType.CASE_MANAGEMENT)
+            .startDate(LocalDateTime.of(2020, Month.FEBRUARY, 1, 11, 11, 11))
+            .judgeAndLegalAdvisor(JudgeAndLegalAdvisor.builder()
+                .judgeTitle(JudgeOrMagistrateTitle.HER_HONOUR_JUDGE)
+                .judgeLastName("Smith")
+                .judgeEmailAddress("test@test.com")
+                .build())
+            .build());
+
+        HearingOrder cmo = HearingOrder.builder()
+            .type(AGREED_CMO)
+            .title("Agreed CMO discussed at hearing")
+            .order(DocumentReference.builder().binaryUrl("/54321").build())
+            .build();
+
+        HearingOrder c21 = HearingOrder.builder()
+            .type(C21)
+            .title("Test order")
+            .order(DocumentReference.builder().binaryUrl("/99999").build())
+            .build();
+
+        CaseData caseData = CaseData.builder()
+            .id(100L)
+            .respondents1(wrapElements(Respondent.builder()
+                .party(RespondentParty.builder()
+                    .lastName("Smith")
+                    .build())
+                .build()))
+            .hearingDetails(List.of(hearingBooking))
+            .ordersToBeSent(List.of(element(cmo), element(c21)))
+            .lastHearingOrderDraftsHearingId(hearingBooking.getId())
+            .build();
+
+        DraftOrdersApproved event = new DraftOrdersApproved(caseData, List.of(cmo, c21));
+
+        underTest.sendNotificationToAdminAndLA(event);
+
+        assertThat(response())
+            .hasSubject("New orders issued, Smith")
+            .hasBody(emailContent()
+                .line("New orders have been issued for:")
+                .line()
+                .callout("Smith, case management hearing, 1 February 2020")
+                .line()
+                .line("The orders are:")
+                .line()
+                .callout("Agreed CMO discussed at hearing\nTest order")
+                .line()
+                .h1("Next steps")
+                .line()
+                .line("You should now check the orders to see if you have any directions and compliance dates.")
+                .line()
+                .line("You can review the orders by:")
+                .list("signing into http://fake-url/cases/case-details/100#Orders")
+                .list("using these links: \n\n* http://fake-url/54321\n* http://fake-url/99999")
+                .line()
```

Comment:
@samuelzerozero any ideas for something better than this? didn't have much time to look into EmailContent class - template: https://www.notifications.service.gov.uk/services/12f756df-f01d-4a32-a405-e1ea8a494fbb/templates/632ea051-0ab0-447d-9d6c-54118b04a5c1